packages/core/src/components/Handle/index.tsx

```diff
@@ -66,9 +67,11 @@ const Handle = forwardRef<HTMLDivElement, HandleComponentProps>(
       onConnect?.(edgeParams);
     };
 
-    const onMouseDownHandler = (event: ReactMouseEvent<HTMLDivElement>) => {
-      if (event.button === 0) {
-        handleMouseDown({
+    const onPointerDown = (event: ReactMouseEvent<HTMLDivElement> | ReactTouchEvent<HTMLDivElement>) => {
+      const isMouse = isMouseEvent(event);
+
+      if ((isMouse && event.button === 0) || 'touches' in event) {
```

Comment:
Here, constant zero might be considered as a `value`; while it is used to denote a `MouseEvent` button value (0 -> no button, 1 -> primary and so on) (https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/buttons)

So, in case if we are also having `event.button === 1` and so on; we should replace with meaningful constant vars here.

Reply:
thanks for your input here. button === 0 means left click here

Reply:
Ohhh, interesting. Saw the changes made; thanks!